packages/augur-ui/package.json

```diff
@@ -101,27 +101,34 @@
     "shelljs": "0.8.3",
     "shifty": "2.6.1",
     "speedomatic": "2.1.4",
-    "trezor-connect": "5.0.34"
+    "trezor-connect": "5.0.31",
+    "webpack": "4.19.1",
+    "webpack-dev-middleware": "3.3.0"
   },
   "devDependencies": {
-    "@types/expect-puppeteer": "2.2.4",
-    "@types/jest": "23.3.10",
-    "@types/jest-environment-puppeteer": "2.2.1",
-    "@types/puppeteer": "1.11.1",
-    "augur-tools": "6.0.0",
-    "autoprefixer": "7.2.6",
+    "@types/expect-puppeteer": "3.3.0",
+    "@types/jest": "^23.3.10",
+    "@types/jest-environment-puppeteer": "^2.2.1",
+    "@types/puppeteer": "^1.6.4",
+    "autoprefixer": "9.1.5",
+    "babel-core": "7.0.0-bridge.0",
     "babel-eslint": "9.0.0",
-    "babel-plugin-istanbul": "4.1.6",
-    "babel-plugin-transform-react-remove-prop-types": "0.4.20",
-    "chai": "4.2.0",
-    "chai-subset": "1.6.0",
+    "@babel/core": "7.2.0",
```

Comment:
These have all been hoist. I imagine this file shouldn't change much save for the script section